app/views/paginable/users/_index.html.erb

```diff
@@ -1,43 +1,78 @@
-<table class="table tablesorter table-hover table-bordered">
-  <thead>
-    <tr>
-      <th><%= _('Name') %></th>
-      <th><%= _('Email address') %>&nbsp;<%= paginable_sort_link('email') %></th>
-      <th class="text-center"><%= _('Last logged in') %>&nbsp;<%= paginable_sort_link('last_sign_in_at') %></th>
-      <th class="text-center"><%= _('How many plans?') %></th>
-      <th class="text-center"><%= _('Privileges') %></th>
-    </tr>
-  </thead>
-  <tbody>
-    <% scope.each do |user| %>
-      <% if !user.nil? then%>
-        <tr>
-          <td>
-            <% if !user.name.nil? %>
-                <%= user.name(false) %>
-            <% end %>
-          </td>
-          <td>
-              <%= user.email %>
-          </td>
-          <td class="text-center">
-            <% if !user.last_sign_in_at.nil? %>
-                <%= l user.last_sign_in_at.to_date, :formats => :short %>
-            <% end %>
-          </td>
-          <td class="text-center">
-            <% if !user.roles.nil? %>
-                <%= user.roles.length %>
-            <% end %>
-          </td>
-          <td class="text-center">
-            <% unless current_user == user %>
-                <% b_label = _('Edit')%>
-                <%= link_to b_label, admin_grant_permissions_user_path(user)%>
-            <% end %>
-          </td>
-        </tr>
-      <% end %>
-    <% end %>
-  </tbody>
-</table>
\ No newline at end of file
+<div class="row">
```

Comment:
This partial view get rendered within the paginable layout therefore you only have to write code here that display your data in this case the table. There are several things to change here:
- Please, remove lines 1 to 11. The header of this view should be modified at app/views/users/admin_index.html.erb which is the view rendered when users#admin_index action is requested.
- Please, remove lines 15 to 23 since the search functionality is rendered within the paginable layout and this views stays DRI.
- The header columns must call paginable_sort_link helper in order to enable the sorting functionality passing the table column as an string argument (see other examples at app/views/paginable/**/*.html.erb.